analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/SymbolicRuleCheck.cs

```diff
@@ -52,7 +52,7 @@ public void Init(SonarAnalysisContext sonarContext, SonarSyntaxNodeReportingCont
     protected void ReportIssue(IOperation operation, params object[] messageArgs) =>
         ReportIssue(operation, null, messageArgs);
```

Comment:
```suggestion
        ReportIssue(operation, messageArgs);
```

Reply:
I feel that this is a potential cause for confusion.
Do you think it's a good idea to change a bit the design in these overloads?

Reply:
From my experience, this is the usual way to deal with overloads calling each other. Do you have a suggestion on how to make it less confusing?

Reply:
I think params is what creates the potential confusion here - in combination with the default parameter.
Maybe I'd remove the default parameter for now and set it explicitly to null where we call it.
It will change anyways as we'll need to provide additional locations in any case.
WDYT?